### ISSUE TITLE
App stuck after answering the call with incoming call screen.

### ISSUE THREAD

**Issue by dhavalpanchani** [react-native-webrtc/react-native-callkeep#546]
# Bug report

- [ ] I've checked the [example](https://github.com/react-native-webrtc/react-native-callkeep/tree/master/example) to reproduce the issue.

- Reproduced on:
 - [ ] Android

## Steps to Reproduce

![Screenshot 2022-03-17 at 1 53 07 PM](https://user-images.githubusercontent.com/25957550/158767281-ddf24dc7-2ca0-4db3-8d86-c9e46cc20e88.png)

## Versions
    - Callkeep:
    - React Native:
    - iOS:
    - Android:
    - Phone model: 

## Logs

```
Paste here
```

**Comment by trandinhhiep789** [comment]
Yes, I have tried these methods(endCall(), endAllCall()) to dismiss the call in "answerCall" listener but incoming UI is still displayed too. Have you solved this problem yet?

**Comment by juanxog** [comment]
@trandinhhiep789 same here! did you find some solution ?

**Comment by iim-norse** [comment]
same here :( any solution ?

**Comment by Irfanwani** [comment]
In my case, answerCall eventlistener is not added sometimes, and this mainly happens in production mode on android.

**Comment by Irfanwani** [comment]
Isthere any way to check whether the eventlistener is added or not. I want to do something like, if eventlistener fails to add, i will try again until it suceeds, but as eventlisteners don't throw any error, i don't know any way to do so.
